airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -62,13 +67,15 @@ def __init__(
         self,
         connections_prefix: str = 'airflow/connections',
         variables_prefix: str = 'airflow/variables',
+        configs_prefix: str = 'airflow/configs',
```

Comment:
This should probably just be config, not configs -- as config is both singular and plural, at least in British English. (Configurations/configs) sounds odd to my ears.

Reply:
fixed in https://github.com/apache/airflow/pull/9645/commits/35d75e5a52d413ecb97d935060011472625bac01